src/activeLighting.js

```diff
@@ -91,50 +91,50 @@ class ATL {
         Hooks.on("updateActiveEffect", async (effect, change, options, userId) => {
             if (game.userId !== userId) return;
```

Comment:
For the proposed `getEffects` to work, we need to make sure this is an active effect on the actor and not an item. That check is done later in `ATL.applyEffects` but we need to check here first.
```suggestion
            if (game.userId !== userId || !(effect.parent instanceof Actor)) return;
```